tensorflow/stream_executor/dnn.h

```diff
@@ -822,10 +822,22 @@ class ProfileResult {
 //  algorithm: the primary algorithm that should be used.
 //  algorithm_no_scratch: a secondary algorithm that should be used, if the
 //    the allocation for the scratch memory fails.
+//  scrach_size: specify the size of scratch memory in bytes needed for the
+//    algorithm used
```

Comment:
Missing '.'

Reply:
thanks for spotting this. it's been fixed.